internal/storage/storageutil/object_attrs_test.go

```diff
@@ -0,0 +1,102 @@
+package storageutil
+
+import (
+	"crypto/md5"
+	"testing"
+	"time"
+
+	"cloud.google.com/go/storage"
+	. "github.com/jacobsa/ogletest"
+	storagev1 "google.golang.org/api/storage/v1"
+)
+
+const TestBucketName string = "gcsfuse-default-bucket"
+const TestObjectName string = "gcsfuse/default.txt"
+const Test string = "test"
+
+func TestObjectAttrsToBucketObject(t *testing.T) { RunTests(t) }
+
+type objectAttrsTest struct {
+}
+
+func init() { RegisterTestSuite(&objectAttrsTest{}) }
+
+func (t objectAttrsTest) TestObjectAttrsToBucketObjectMethod() {
+	var attrMd5 []byte
+	Time := time.Now()
+	attrs := storage.ObjectAttrs{
+		Bucket:                  TestBucketName,
+		Name:                    TestObjectName,
+		ContentType:             Test,
```

Comment:
use different value for each attribute. otherwise we will not be able to catch errors like in the main code if someone sets contentLanguage to contentType.

no need to define all of them as constants.

Reply:
fixed